src/pkg/packager/creator/skeleton.go

```diff
@@ -0,0 +1,312 @@
+				return nil, fmt.Errorf(lang.ErrFileExtract, file.ExtractPath, file.Source, err.Error())
+			}
+
+			// Make sure dst reflects the actual file or directory.
+			updatedExtractedFileOrDir := filepath.Join(destinationDir, file.ExtractPath)
+			if updatedExtractedFileOrDir != dst {
+				if err := os.Rename(updatedExtractedFileOrDir, dst); err != nil {
+					return nil, fmt.Errorf(lang.ErrWritingFile, dst, err)
+				}
+			}
+		} else {
+			if err := utils.CreatePathAndCopy(file.Source, dst); err != nil {
+				return nil, fmt.Errorf("unable to copy file %s: %w", file.Source, err)
+			}
+		}
+
+		// Change the source to the new relative source directory (any remote files will have been skipped above)
+		updatedComponent.Files[filesIdx].Source = rel
+
+		// Remove the extractPath from a skeleton since it will already extract it
+		updatedComponent.Files[filesIdx].ExtractPath = ""
+
+		// Abort packaging on invalid shasum (if one is specified).
+		if file.Shasum != "" {
+			if err := utils.SHAsMatch(dst, file.Shasum); err != nil {
+				return nil, err
+			}
+		}
+
+		if file.Executable || utils.IsDir(dst) {
+			_ = os.Chmod(dst, 0700)
+		} else {
+			_ = os.Chmod(dst, 0600)
+		}
+	}
+
+	if len(component.DataInjections) > 0 {
+		spinner := message.NewProgressSpinner("Loading data injections")
+		defer spinner.Stop()
+
+		for dataIdx, data := range component.DataInjections {
+			spinner.Updatef("Copying data injection %s for %s", data.Target.Path, data.Target.Selector)
+
+			rel := filepath.Join(layout.DataInjectionsDir, strconv.Itoa(dataIdx), filepath.Base(data.Target.Path))
+			dst := filepath.Join(componentPaths.Base, rel)
+
+			if err := utils.CreatePathAndCopy(data.Source, dst); err != nil {
+				return nil, fmt.Errorf("unable to copy data injection %s: %s", data.Source, err.Error())
+			}
+
+			updatedComponent.DataInjections[dataIdx].Source = rel
+		}
+
+		spinner.Success()
+	}
+
+	if len(component.Manifests) > 0 {
+		// Get the proper count of total manifests to add.
+		manifestCount := 0
+
+		for _, manifest := range component.Manifests {
+			manifestCount += len(manifest.Files)
+			manifestCount += len(manifest.Kustomizations)
+		}
+
+		spinner := message.NewProgressSpinner("Loading %d K8s manifests", manifestCount)
+		defer spinner.Stop()
+
+		// Iterate over all manifests.
+		for manifestIdx, manifest := range component.Manifests {
+			for fileIdx, path := range manifest.Files {
+				rel := filepath.Join(layout.ManifestsDir, fmt.Sprintf("%s-%d.yaml", manifest.Name, fileIdx))
+				dst := filepath.Join(componentPaths.Base, rel)
+
+				// Copy manifests without any processing.
+				spinner.Updatef("Copying manifest %s", path)
+
+				if err := utils.CreatePathAndCopy(path, dst); err != nil {
+					return nil, fmt.Errorf("unable to copy manifest %s: %w", path, err)
+				}
+
+				updatedComponent.Manifests[manifestIdx].Files[fileIdx] = rel
+			}
+
+			// remove kustomizations
+			updatedComponent.Manifests[manifestIdx].Kustomizations = nil
```

Comment:
I believe this will fail as we are not currently pre building kustomizations for skeletons

Reply:
Yeah I've confirmed this is an issue by publishing a skeleton of 09-composable-packages with zarf built from this pr and the current zarf release. The configmap made from kustomize is missing in that instance

Reply:
Good catch! Fixed and added logic to `func (suite *SkeletonSuite) Test_2_FilePaths()` to verify that kustomize-generated manifests are included in skeleton packages.